src/artifacts_keyring/plugin.py

```diff
@@ -35,9 +35,10 @@ def __init__(self):
                 from dotnetcore2.runtime import get_runtime_path
             except ImportError as e:
                 message = (
-                    "Unable to find full dotnetcore2 runtime path;"
-                    " the tool will attempt with just 'dotnet' in case"
-                    " it is in the user-defined PATH variable. Error: "
+                    "Unable to find dependency dotnetcore2; the tool will"
+                    " attempt to call 'dotnet' directly. If unable to install"
+                    " the dotnetcore2 Python package, please install manually"
```

Comment:
nit: "manually install" reads better than "install manually"

Reply:
```suggestion
                    " the dotnetcore2 Python package, please manually install"
```

Suggestions are a great way to get nits fixed (it's the left-most toolbar button) :)